src/modules/functions.js

```diff
@@ -0,0 +1,95 @@
+// Get the tasks from local storage
+export const getTasks = () => {
+  let tasks;
+
+  if (localStorage.getItem('tasks')) {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  } else {
+    tasks = [];
+  }
+
+  return tasks;
+};
+
+// Add a task to local storage
+export const storeTask = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const taskContainer = document.querySelector('.list__tasks');
+
+export const render = (tasks) => {
```

Comment:
On the [render](https://github.com/Timbar09/To-Do-list/blob/main/src/modules/functions.js#L21) function, I think you could have implemented object restructuring to extract the values of the tasks array keys instead of using the traditional way like "tasks.description".

Reply:
Thanks, @Oscah01

Reply:
Yeah, I think you are right object destructuring is not necessary for this situation.